### PR TITLE
feat: document how to customize redirect url

### PR DESCRIPTION
**Pull request by aeneasr**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

**Review by Benehiko — APPROVED** [review]

**Review by Benehiko — DISMISSED** [review]

File: docs/identities/sign-in/social-sign-in/redirect-url.mdx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+id: redirect-url
+title: Set custom domain as redirect URL
+sidebar_label: Custom domain redirect
+---
+
+Some Social Sign-in Providers such as Google or GitHub show the Redirect URL as part of the sign-in process:
+
+![GitHub showing the redirect URL](redirect-url/callback.png)
+
+By default, Ory Cloud uses a redirect URL that points to the Ory Cloud domain.
```
[the remark this answers is not in the document]
Comment (aeneasr) [reply docs/identities/sign-in/social-sign-in/redirect-url.mdx]: bad copilot 😠

**Review by vinckr — APPROVED** [review]
awesome!

**Comment by aeneasr** [comment]
@vinckr can you please try if it works? :D

**Comment by vinckr** [comment]
I can confirm that it works.

Very important to match the URI everywhere!

File: docs/identities/sign-in/social-sign-in/redirect-url.mdx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+By default, Ory Network uses a redirect URL that points to the Ory Network domain.
+
+:::caution
+
+Changing the `base_redirect_uri` will break previously existing and functional social sign-in connections.
+
+To fix a broken connection, choose the provider you want to fix from the Ory Console social sign-in configuration screen and copy
+the updated redirect URL which now points to your custom hostname. Use the value you copied to update the redirect URL
+configuration of the social sign-in provider.
```
Comment (vinckr) [inline docs/identities/sign-in/social-sign-in/redirect-url.mdx:19]:
```suggestion
To fix a broken connection, choose the provider you want to fix from the Ory Console social sign-in configuration screen and copy
the redirect URL. You need to replace the Ory Network URl with your custom hostname. Use this URL to update the redirect URL
configuration of the social sign-in provider.
```

Right now the URL does not update in the console

**Pull request merged**